test/integration/jupyter_notebooks_test.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+import nbformat
+import pytest
+from nbconvert.preprocessors import ExecutePreprocessor
+from sqlalchemy.engine import Engine
+
+
+@pytest.mark.parametrize(
+    "notebook",
+    [
+        "devtools/inspect-assets.ipynb",
+        "devtools/debug-eia-etl.ipynb",
+        "devtools/debug-ferc1-etl.ipynb",
+        "devtools/debug-harvesting.ipynb",
```

Comment:
These all run pretty quickly now that they're relying on assets generated by Dagster for the most part, and they're prone to getting out of sync w/ the codebase, so I thought it made sense to add them into the tests finally. And also I just wanted to understand how to do that, since @katie-lamb and I are going to try and run some notebooks in CI for the record-linkage project.